Sources/View/Widget/SelectionWidget.swift

```diff
@@ -105,12 +98,7 @@ final class SelectionWidget: AbstractWidget {
         }
 
         tableView.initialSelectedItemIndex = tableView.items.firstIndex { $0.value == value() }
-
-        tableView.shouldMarkCellAction = { option in
-            self.labelField.text == option.label.localized()
-        }
-
-        /// setup search bar
+        tableView.shouldMarkCellAction = { self.selectedValue == $0.value }
```

Comment:
```self```

Reply:
We can't remove because is a closure scope